federatedscope/core/trainers/trainer_simple_tuning.py

```diff
@@ -0,0 +1,70 @@
+import copy
+import torch
+import logging
+import math
+
+from federatedscope.core.auxiliaries.optimizer_builder import get_optimizer
+from federatedscope.core.trainers.torch_trainer import GeneralTorchTrainer
+
+from typing import Type
+
+logger = logging.getLogger(__name__)
+
+
+def wrap_Simple_tuning_Trainer(
+        base_trainer: Type[GeneralTorchTrainer]) -> Type[GeneralTorchTrainer]:
+
```

Comment:
Add some explanations for the purpose of this wrapper function, and the meaning of `Simple_tuning` as it is too general.